File: JAVA/Route.java
import java.util.ArrayList;

public class Route {
    //Propiedades o atributos 
    Integer id;
    ArrayList<Double> start;
    ArrayList<Double> end;

    // Método constructor
    public Route(Integer id, ArrayList<Double> start, ArrayList<Double> end){
        this.id = id;
        this.start = start;
        this.end = end;
    }
    
}

File: PHP/route.php
<?php
    class Route{
        public $id;
        public $start = array();
        public $end = array();

        public function __construct($id,  $start, $end){
            $this->id = $id;
            $this->start = $start;
            $this->end = $end;
        }
    }
?>